### PR TITLE
feat: style improvements

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Adds a pending message in the signing state:

https://github.com/cowprotocol/cowswap/assets/31534717/4ba42b49-6854-46a6-b2a4-0288620dadd2

### CODE REVIEW

File: apps/cowswap-frontend/src/common/pure/TradeLoadingButton/index.tsx [hunk 1]
```diff
@@ -3,31 +3,10 @@ import { useEffect, useState } from 'react'
...
-const CenteredDots = styled(Dots)<{ smaller?: boolean }>`
-  vertical-align: ${({ smaller = false }) => (smaller ? 'normal' : 'super')};
-`
-
-const LongLoadText = styled.span`
-  animation: fadeIn 0.42s ease-in;
-
-  ${fadeIn}
-`
+import { CenteredDots, LongLoadText } from 'modules/trade/pure/TradeConfirmation/styled'
```
Comment (shoom3301) [inline apps/cowswap-frontend/src/common/pure/TradeLoadingButton/index.tsx:9]: Common things should not depend on modules.
Would be event better to move the component into `@cowprotocol/ui`

**Comment by elena-zh** [comment]
Hey @fairlighteth, great job!
Question: there are loaders instead of the message in the Classic eth-flow modal. Is there possibility to fix this in the current PR or it is better to create a separate issue for this? 
approving
classic eth
Screenshot_1

Thanks!

**Comment by fairlighteth** [comment]
@elena-zh Addressed all points in this PR:


https://github.com/cowprotocol/cowswap/assets/31534717/6244c8e7-a8c5-4889-8d45-588657fb8484


https://github.com/cowprotocol/cowswap/assets/31534717/2843b536-fdda-4094-87ad-b41c60abf0a6

File: apps/cowswap-frontend/src/common/pure/TradeLoadingButton/index.tsx (hunk 1 above)
Comment (fairlighteth) [reply apps/cowswap-frontend/src/common/pure/TradeLoadingButton/index.tsx]: @shoom3301 addressed this now

**Comment by elena-zh** [comment]
Hey @fairlighteth , great, thank you!
One proposal from my side is to change 'Sing' to 'Confirm' word for the Classic eth-flow modals (as these are onchain TXs). WDYT? :)
confirm

**Review by elena-zh — APPROVED** [review]
Great, thank you!

**Pull request merged**